style.css

```diff
@@ -1,114 +1,215 @@
+.row{
+    display: flex;
```

Comment:
In the CSS file, it seems like you are using 4 spaces. It should be 2 spaces ideally.